### DISCUSSION TITLE
How do i disable esmbot?

### DISCUSSION THREAD

**Discussion by N1GHTM4R** [esmBot/esmBot#469] [General]
It's annoying when I'm talking with a friend, then make a stupid joke and accidentally send a random photo from my camera roll. is there any way to disable this?

**Comment by TheEssem** [comment]
If this is happening in a server where the bot has been invited, it should be possible for an admin to enable/disable commands from Server Settings -> Integrations; similarly, if this is happening in a server where the bot has not been invited, then a server admin can disable the "Use External Apps" role permission.

If this is happening in DMs, then to my knowledge it's not possible at the moment to disable applications there specifically (besides removing the app from your account altogether); this would have to be done on Discord's end, not mine.

**Reply by N1GHTM4R** [reply]
yeah, i just had to deauthorize it. cheers
